scipy/spatial/tests/test_hausdorff.py

```diff
@@ -133,6 +133,15 @@ def test_invalid_dimensions(self):
          [(0,1), (0,0)],
          1,
          (0.0, 0, 1)),
+        # gh-11332 cases with a Generator
+        ([(0,0)],
+         [(0,1), (0,0)],
+         np.random.default_rng(0),
+         (0.0, 0, 1)),
+        ([(0,0)],
+         [(0,1), (0,0)],
+         np.random.default_rng(1),
+         (0.0, 0, 1)),
```

Comment:
Where do the reference values come from?

Reply:
Two things:

1) It is a direct copy paste of an already-tested case above, with the adjustment of using a `Generator` instead of an integer seed.
2) The expected value can be deduced by straightforward reasoning--the "maximum minimum" distance between `u` and `v` must be 0 since there's an identical point in the two "curves."

Reply:
And note that this is asymmetric, in the reverse order (`v` to `u`) there are two points in `v` and the "maximum minimum" becomes `1`.